wtp/itests/org.jboss.tools.wtp.runtimes.tomcat.itests/src/org/jboss/tools/wtp/runtimes/tomcat/itests/AbstractTomcatDetectionTest.java

```diff
@@ -121,21 +121,21 @@ protected String toString(Object[] runtimes) {
     }
 
     @SuppressWarnings("restriction")
-	private static void checkJRE7Availability() {
-      IExecutionEnvironment se7Env = EnvironmentsManager.getDefault().getEnvironment(IExecutionEnvironmentConstants.EXEC_ENV_JavaSE17);
-      System.out.println("jre7 home is " + JRE_7_HOME);
-      if (JRE_7_HOME != null && !JRE_7_HOME.startsWith("${")) {
-        assertTrue("JRE7 home " + JRE_7_HOME + " does not exist", new File(JRE_7_HOME).exists());
-        IVMInstall foundOrCreated = JREUtils.findOrCreateJRE(new Path(JRE_7_HOME));
+	private static void checkJRE8Availability() {
+      IExecutionEnvironment se8Env = EnvironmentsManager.getDefault().getEnvironment(IExecutionEnvironmentConstants.EXEC_ENV_JavaSE18);
+      System.out.println("jre8 home is " + JRE_8_HOME);
+      if (JRE_8_HOME != null && !JRE_8_HOME.startsWith("${")) {
+        assertTrue("JRE7 home " + JRE_8_HOME + " does not exist", new File(JRE_8_HOME).exists());
```

Comment:
please fix typo here
```suggestion
        assertTrue("JRE8 home " + JRE_8_HOME + " does not exist", new File(JRE_8_HOME).exists());
```

Reply:
Thanks! Fixed.